app/controllers/ctc/portal/verification_attempts_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Ctc::Portal::VerificationAttemptsController < Ctc::Portal::BaseAuthenticatedController
+  layout "intake"
+  helper_method :prev_path, :illustration_path, :illustration_folder
```

Comment:
i need to define these methods as helper methods so that i can use the a shared layout without directly inheriting from questions controller.

Reply:
it lets you use them as methods in the template. the intake layout requires that these methods are defined in order to display the back button and icons